modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/portal_pop_up.ftl

```diff
@@ -10,7 +10,7 @@
 	<@liferay_util["include"] page=top_head_include />
 </head>
 
-<body class="portal-popup ${css_class}">
+<body class="dialog-iframe-popup dialog-with-footer portal-popup ${css_class}">
```

Comment:
Hey @markocikos, I went through your explanation of this at https://github.com/wincent/liferay-portal/pull/250. 

Changing the theme for this makes me uneasy... it should probably be okay for `dialog-iframe-popup`, but I'm not so sure about `dialog-with-footer`... Doing it here also means that any other theme that overrides `portal_pop_up` might not have those classes or would require an update. 

Moving forward we should try to steer clear of any change in themes for basic functionality we need across them since they can't be relied upon. 

Since this has been lingering for a while, I'd just push to keep it for now and try to rethink the solution as soon as we get this merged.

Reply:
I found a bug that adding `dialog-with-footer` everywhere would cause, working on fix or an alternative solution.